doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -472,15 +472,21 @@ Enabling and disabling notations
    :n:`@string`
       Notations to enable or disable. :n:`@string` can be a single
       token in the notation such as "`->`" or a pattern that matches
-      the notation. See :ref:`locating-notations`.
+      the notation. See :ref:`locating-notations`. If no :n:`{? :=
+      @one_term }` is given, the variables of the notation can be
```

Comment:
```suggestion
      the notation. See :ref:`locating-notations`. If no
      :n:`{? := @one_term }` is given, the variables of the notation can be
```

line breaks in `:n:` show up in the output, like this:
![image](https://user-images.githubusercontent.com/1253341/206884005-3b89bd0c-bc31-4feb-b30f-8413adc7917c.png)